frontend_v2/pages/newcountry.tsx

```diff
@@ -55,16 +62,15 @@ export default function NewTree({initialTree}) {
                 <div>
                 
                     <div> 
-                        <AddTreeForm 
+                        <AddCountryForm 
                             onSubmit={ async (data) => {
                                 try {
-                                    await saveTree(data);
-                                    setTree([...trees, data]);
+                                    await saveCountry(data);
+                                    setCountry([...country, data]);
                                 } catch (err) {
                                     console.log(err)
                                 }
```

Comment:
You might want to move this into a function instead of directly handling the `onSubmit` here